packages/x-components/src/wiring/events.types.ts

```diff
@@ -143,6 +143,11 @@ export interface XEventsTypes
    * * Payload: The scroll id which has scrolled to top.
    */
   UserClickedScrollToTop: string;
+  /**
+   * The user has open X with the API.
```

Comment:
In order to have the same time as the rest of descriptions it should be: `The user opened X programmatically`. Notice that I also suggest to replace with the API with programmatically as the developer user can configure it to be emitted from any part of the app.